client/packages/reports/src/ListView/ListView.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Grid } from '@openmsupply-client/common';
+import { AppBarButtons } from './AppBarButton';
+import { SidePanel } from './SidePanel';
+import { StockAndItemWidget } from './StockAndItemWidget';
+import { useReport } from '@openmsupply-client/system';
+import { ExpiringWidget } from './ExpiringWidget';
+
+export const ListView = () => {
+  const { data } = useReport.document.list({});
+  const stockAndItemReports = data?.nodes?.filter(
+    report => report?.subContext === 'StockAndItems'
```

Comment:
Probably want an enum of sub-contexts so we can reliability search for the correct sub-context...needs change in mSupply? 🤔

Reply:
I think we would need another column on report, we can get to it when we transfer program report from dispensary menu to here. Since sub context in those reports seems to be linked to the program (although I can't find anywhere in code where sub context for reports is used ¯\_(ツ)_/¯)